specs/networking/p2p-interface.md

```diff
@@ -414,12 +434,18 @@ Response Content:
 )
 ```
 
-Requests blocks by their block roots. The response is a list of `BeaconBlock` with the same length as the request. Blocks are returned in order of the request and any missing/unknown blocks are left empty (SSZ null `BeaconBlock`).
+Requests blocks by their block roots. The response is a list of `BeaconBlock` whose length is less than or equal to the number of requested blocks. It may be less in the case that the responding peer is missing blocks.
 
-`RecentBeaconBlocks` is primarily used to recover recent blocks (ex. when receiving a block or attestation whose parent is unknown).
+`BeaconBlocksByRoot` is primarily used to recover recent blocks (e.g. when receiving a block or attestation whose parent is unknown).
+
+The request MUST be encoded as an SSZ-field.
```

Comment:
Is this still correct? The chunks are ssz containers now, `BeaconBlock`. I.e. it's a series of streamed containers rather than a ssz-field `[]BeaconBlock`.

Reply:
Oops, i misread request/response. Please disregard